tap-gui/customize/customize-telemetry.hbs.md

```diff
@@ -0,0 +1,28 @@
+# Customize the Tanzu Application Platform GUI telemetry collection
+
+This section describes how to customize telemetry collection for the Tanzu Application Platform GUI portal.
+
+## <a id="telemetry-customizing"></a> Customize organization ID
+
+By default, each instance of Tanzu Application Platform GUI is assigned to a random organization ID to ensure that your sensitive information is not revealed. 
+
+However, you may choose to customize your organization ID and self-identify. That would allow VMware to observe account-level telemetry (please note that all personally-identifyable information would remain anonymized in any case): frequency of portal usage, most popular functionality, etc. 
```

Comment:
Maybe we should explain how PII would be anonymized?

Something like
> The ORGANIZATION-NAME will be hashed in order to prevent VMware from identifying you by the value. Thus, ORGANIZATION-NAME need not identify your organization, but should be unique and static across an instance of Tanzu Application Platform GUI so that across refreshes of the Tanzu Application Platform database your ORGANIZATION-NAME remains the same.

It's good the way it is though, that's more of a nit than anything

Reply:
Now that the PR is already merged, I'll work this in myself. It's good information and it can be blended with what's already there.